components/generic_modal.tsx

```diff
@@ -139,22 +139,20 @@ export default class GenericModal extends React.PureComponent<Props, State> {
                 <Modal.Header
                     closeButton={true}
                 />
-                <form>
```

Comment:
@julmondragon this is the <form> tag belonging to the generic modal that is causing you issues when using your new input widget. For what I can see it is safe to remove, actually, I am removing it in this PR and besides all tests passing, I checked some places where the generic modal is used and seems to continue working fine, but I want to confirm if it was just a html-tag leftover or was set here intentionally. Wondering if @devinbinnie remembers if there was a reason for it?

Reply:
I don't explicitly remember if there was, but my guess was to follow semantic HTML for accessibility reasons. Just spitballing here though.

Reply:
@pablovelezvidal can you list the modals that use this `GenericModal` so @stevemudie can have at pass at them in this PR and make sure they are working as expected still?

Reply:
Thanks @marianunez, Sure:

1. Confirm license removal modal
2. Enterprise Edition information agreement modal (licensing pages in starter mode have some links to show a modal witht the EE agreement info)
3. Upload license modal
4. No internet connection modal (announcement bar shows info about renewing the license, when you click and somehow there is no internet access that modal is presented)
5. Collapse/Reply Threads Beta modal (I think this is in beta state and not sure how to get it shown)
6. Collapse/Reply Threads modal
7. Custom Status Modal
8. Delete Category Modal
9. DnD Custom timepicker modal
10. Edit category modal
11. Product Notices modal
12. Remove next steps modal
13. Trial Benefits modal

@stevemudie let me know if you have any question or  face any issues trying to validate this.

Reply:
ok, we re-reviewed all these modals and everything seems to be working fine. Thanks @stevemudie for the help.